appveyor.yml

```diff
@@ -0,0 +1,34 @@
+# https://www.appveyor.com/docs/appveyor-yml
+
+version: "{build}"
+
+clone_depth: 10
+
+environment:
+  PLATFORM: "x64"
+  matrix:
+    # check the current Node.js release and the minimum supported one
+    - NODEJS_VERSION: "10"
+    - NODEJS_VERSION: "6"
```

Comment:
Why not add node version 8 ?

Reply:
Because @hzoo didn't want to add it.

Reply:
Any special reason, travis ci has node 8. And node 8 is lts release.